pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -172,6 +173,18 @@ type CSINodeInfo interface {
 	GetCSINodeInfo(nodeName string) (*storagev1beta1.CSINode, error)
 }
 
+var _ CSINodeInfo = &CachedCSINodeInfo{}
+
+// CachedCSINodeInfo implements CSINodeInfoInfo
+type CachedCSINodeInfo struct {
```

Comment:
nit: the API object has been renamed to `CSINode` (the `Info` is omitted)

Reply:
I wanted to be consistent with the other few interfaces defined in the same file (all suffixed with info). I agree we probably don't need any of that, but as I mentioned I would like to keep the code consistent while working on removing all of them completely.